src/webview/js/js.js

```diff
@@ -618,7 +618,18 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
   }
 });
 
-const getHtmlNodeType = (target: Element) => target.matches('.header') ? 'header' : 'message';
+const getHtmlNodeType = (target: Element) => {
+  if (target.matches('.header')) {
+    return 'header';
+  } else if (
+    target.matches('.reaction-list')
```

Comment:
Examine how we use `element.closest('body > *');` for similar purposes and think if you can do the same here.

Reply:
Done, now I am using `.closest` 👍